test/library/standard/DataFrames/DataFrames.chpl

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2018 Cray Inc.
+ * Copyright 2004-2019 Cray Inc.
```

Comment:
I believe we typically skip checking files in the `test/` directory.  Did you use `util/test/checkCopyrights.bash`?  If so that might indicate a bug in that script, since in the past it has avoided them.

Reply:
I can remove them, but is there a reason that the copyright on the tests isn't kept up to date? Just too much overhead?

Reply:
Partially overhead, partially that tests are not particularly interesting intellectual property.  We have a lot of files in the test directory in support of the tests themselves which are almost trivial, and many tests come from users when they have discovered an issue that needs to be reproduced.  Additionally, we only distribute a select few of them that have been curated as part of our release (the release/examples directory), so we aren't really laying claim to the rest.

@bradcray should correct me if I am wrong, or add additional insight that I may have missed

Reply:
If we were to write our own benchmark suite to publish independently of the Chapel release, that would require its own copyright (and likely its own repository and distribution), but so far that has not been something we've pursued

Reply:
We don't generally copyright tests because of the maintenance burden and noise it would add, combined with the fact that we don't really think of tests as holding our IP.  I think that this one got the standard header because the intention was to integrate it into the modules/ directory eventually.  I don't have a problem with updating it, nor with leaving it alone.